alerting/src/main/kotlin/org/opensearch/alerting/transport/TransportDocLevelMonitorFanOutAction.kt

```diff
@@ -575,6 +575,11 @@ class TransportDocLevelMonitorFanOutAction
 
             // Before the "|" is the doc id and after the "|" is the index
             val docIndex = it.key.split("|")
+            val additionalFields = this.fetchDocForFinding(
+                docIndex[1],
+                docIndex[0],
+                monitor.metadataForFindings!!
```

Comment:
do we need version check here for serde?
what happens in upgrade or blue green scenarios?